packages/storybook/stories/ChatComposite/CustomDateTimeFormat.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useState, useEffect } from 'react';
+import { SingleLineBetaBanner } from '../BetaBanners/SingleLineBetaBanner';
+import { COMPOSITE_FOLDER_PREFIX, compositeExperienceContainerStyle } from '../constants';
+import { defaultChatCompositeHiddenControls, controlsToAdd } from '../controlsUtils';
+import { compositeLocale } from '../localizationUtils';
+import { getDocs } from './ChatCompositeDocs';
+import { ContosoChatContainer } from './snippets/Container.snippet';
+import {
+  ChatCompositeSetupProps,
+  ConfigHintBanner,
+  addParrotBotToThread,
+  createThreadAndAddUser
+} from './snippets/Utils';
+
+const messageArray = [
+  'Welcome to an example on how to add powerful customizations to the ChatComposite',
+  'By following this example, Contoso can customize the date time format to whatever they want!',
+  'Have fun!'
+];
+
+const CustomDateTimeFormatStory = (args, context): JSX.Element => {
+  const {
+    globals: { locale }
+  } = context;
+  const [containerProps, setContainerProps] = useState<ChatCompositeSetupProps>();
+
+  const onDisplayDateTimeString = (messageDate: Date): string => {
+    let hours = messageDate.getHours();
+    let minutes = messageDate.getMinutes().toString();
+    let month = (messageDate.getMonth() + 1).toString();
+    let day = messageDate.getDate().toString();
+    const year = messageDate.getFullYear().toString();
+
+    if (month.length === 1) {
+      month = '0' + month;
+    }
+    if (day.length === 1) {
+      day = '0' + day;
+    }
+    const isAm = hours < 12;
+    if (hours > 12) {
+      hours = hours - 12;
+    }
+    if (hours === 0) {
+      hours = 12;
+    }
+    if (minutes.length < 2) {
+      minutes = '0' + minutes;
+    }
+    return (
+      'TimeStamp: ' +
+      year +
+      '-' +
+      month +
+      '-' +
+      day +
+      ', ' +
+      hours.toString() +
+      ':' +
+      minutes +
+      ' ' +
+      (isAm ? 'a.m.' : 'p.m.')
+    );
+  };
+
+  useEffect(() => {
+    const fetchToken = async (): Promise<void> => {
+      if (args.userId && args.token && args.botId && args.botToken && args.endpointUrl && args.displayName) {
+        const newProps = await createThreadAndAddUser(args.userId, args.token, args.endpointUrl, args.displayName);
+        await addParrotBotToThread(
+          args.token,
+          args.botId,
+          args.botToken,
+          args.endpointUrl,
+          newProps.threadId,
+          messageArray
+        );
+        setContainerProps(newProps);
+      } else {
+        setContainerProps(undefined);
+      }
+    };
+    fetchToken();
+  }, [args.userId, args.token, args.botId, args.botToken, args.endpointUrl, args.displayName]);
+
+  const strings = compositeLocale(locale)?.component.strings ?? COMPONENT_LOCALE_EN_US.strings;
+  const compositeStrings = compositeLocale(locale)?.strings ?? COMPOSITE_LOCALE_EN_US.strings;
+  const onDisplayDateTimeStringLocale =
```

Comment:
Why this fallback? We don't set the datetime format via the storybook locale at all. So let's directly assign the local function.